recipe/0001-use-certifi.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/pyppeteer/chromium_downloader.py b/pyppeteer/chromium_downloader.py
+index 8e9e707..4f6bd12 100644
+--- a/pyppeteer/chromium_downloader.py
++++ b/pyppeteer/chromium_downloader.py
+@@ -68,14 +68,9 @@ def download_zip(url: str) -> BytesIO:
+     """Download data from url."""
+     logger.warning('Starting Chromium download. ' 'Download may take a few minutes.')
+ 
+-    # Uncomment the statement below to disable HTTPS warnings and allow
+-    # download without certificate verification. This is *strongly* as it
+-    # opens the code to man-in-the-middle (and other) vulnerabilities; see
+-    # https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
+-    # for more.
+-    # urllib3.disable_warnings()
+-
+-    with urllib3.PoolManager() as http:
++    import certifi
++
++    with urllib3.PoolManager(cert_reqs='CERT_REQUIRED', ca_certs=certifi.where()) as http:

```

Comment:
@kiwi0fruit look ok?

Reply:
I don't know if this patch works (modifies the file in a right way) but that's the gist of the modification to make in it. May be we need regex patch here too like in the pyee?

Reply:
>  May be we need regex patch here too like in the pyee?

My memory fails me: what patch would that be? Generally, the bog-standard `.patch` files, applied well before any scripts start hacking, seem to be more effective over time, even though they "break" more frequently (that's a good thing, in this case).